pkg/cli-runtime/validation/enum.go

```diff
@@ -35,7 +35,7 @@ func Enum(input string, field string, validOptions []string) FieldErrors {
 
 func EnumInvalidValue(input interface{}, field string, validOptions []string) FieldErrors {
 	return FieldErrors{
-		k8sfield.Invalid(k8sfield.NewPath(field), input, fmt.Sprintf("Supported formats are %v", strings.Join(validOptions, ", "))),
+		k8sfield.Invalid(k8sfield.NewPath(field), input, fmt.Sprintf("supported values are %v", strings.Join(validOptions, ", "))),
```

Comment:
This message was introduced when we were adding `json` and `yaml` as valid formats to output workload. The message should be more generic, not use the word `formats` because it won't make sense for something like this new flag, where values are `replace` and `merge`. That's why I changed it but we can double check with design

Reply:
@heyjcollins

Reply:
Using `values` rather than `formats` is a sound decision.
Good call @warango4